Portfolio: details popup window

jovasal opened the pull request:
<h1>Portfolio: details popup window</h1>

1️⃣ You should implement the following interactions:

- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.

2️⃣ In order to associate each project with the popup window details, you must refactor the project section:

- You need to use a JavaScript array to store all of the information for all projects.
- For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
- After that, you could implement the popup window.

3️⃣ You should implement popups for both mobile and desktop screen sizes:

-Mobile popup

- Desktop popup

4️⃣ You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using figma

jovasal committed:
clone a menu

clone

jovasal committed:
first card

first card

jovasal committed:
first done

first done

jovasal committed:
class for each card

cards

jovasal committed:
fourd mobile cards are working

ok

jovasal committed:
changes in card one

card one

Nessrine88 committed:
Add event listeners for each popup button01

jovasal committed:
picture

picture

jovasal committed:
done poppup

popup

jovasal committed:
linters

linters

jovasal committed:
linters

linters

jovasal committed:
eslint

eslint

jovasal committed:
changes linters

linters

Nessrine88 committed:
update linters

1GORDON reviewed (commented):
##
## Hi @jovasal. and @Nessrine88 ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [ ] Kindly remove the live demo from your readme file since this project is not yet deployed.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jovasal committed:
Live DEMO removed

Live DEMO was removed.

Strangeal commented on script.js:
```diff
@@ -0,0 +1,130 @@
+
+button2.addEventListener('click', () => {
+
+});
```
- [x] Great job on implementing the hamburger menu as specified in the project requirement! 👏 However, there is a minor UI issue 🐞 that we need to address and fix to help create a good user experience. 
   - [x] Kindly consider removing these `event` listeners from your codebase as they are not serving any purpose. Having unused code adds unnecessary complexity and potential performance overhead. Removing these unused event listeners will help improve code clarity and maintainability.

jovasal replied:
Thank you so much!!

I removed:

button1.addEventListener('click', () => {

});

button2.addEventListener('click', () => {

});

Code is working.

Strangeal reviewed (changes requested):
Hi @jovasal, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![Alt Text](https://media.giphy.com/media/t8YPf3ACKeouLzEIvr/giphy.gif)
## Highlights
Correct use of GitHub Flow ✅
Well-structured files ✅
All linter checks are passing ✅
Descriptive PR title with a great summary ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jovasal committed:
remove AssEvenListener

button1.addEventListener('click', () => {

});

button2.addEventListener('click', () => {

});

removed

jovasal committed:
buttons were deleted

const button1 = document.getElementById('button1');
const button2 = document.getElementById('button2');

jovasal committed:
error deleted

frempongdev commented on index.html:
```diff
@@ -1,244 +1,465 @@
-          <img src="./images/snap4.png" alt="Project number 2 image">
+      </section>
+
+      <section class="works" id="works">
+        <div class="card card-desk">
+          <img
+            src="./images/Snapshoot Portfolio.png"
+            alt="work-image-project"
+            class="image-mob"
+          />
+          <img
+            class="desktop-image"
+            src="./images/Snapshoot Portfolio desk.png"
+            alt=""
+          />
+
+          <div class="card-content-mob card-content-deskt card-hero">
+            <h2 class="tonic ton">TONIC</h2>
+            <ul class="dates dated-d">
+              <li>CANOPI</li>
+              <li><img src="./images/bullet.svg" alt="counter" /></li>
+              <li class="stack-backend">Back End Dev</li>
+              <li><img src="./images/bullet.svg" alt="counter" /></li>
+              <li class="stack-backend">2015</li>
+            </ul>
+            <p class="paragraph pa">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="stacks">
+              <li class="stack-over">HTML</li>
+              <li class="stack-over">CSS</li>
+              <li class="stack-over">Javascript</li>
+            </ul>
+            <!-- <a href="#" class="btn">See Project</a> -->
+            <button class="btn" id="popupButton1">See Project</button>
+          </div>
```
Great job so far implementing the popup. 👍🏿  There are some few changes I would suggest to complete this project. ♻️ 

[ OPTIONAL ]
- [ ] Kindly remove the sections block from your HTML file as it is required for you to generate it dynamically as the page loads. 🗑️ 

[ OPTIONAL ]
- [ ] Kindly ensure that your popup is the only active part of your page when in view as it is a good User experience practice. Currently you main page seems to have an active scroll when your popup is in view. 
![aboutgsg](https://github.com/jovasal/jovasal.github.io/assets/117144028/8f73a713-7784-4a73-a22c-c87e8c4c5a44)


[ OPTIONAL ]
- [ ] Kindly make sure that your design looks similar to the given Figma template as that is the core of this section of your learning. Paying attention to details can go a long way to help you satisfy your future clients. 🧰 

### Your design ⬇️
![image](https://github.com/jovasal/jovasal.github.io/assets/117144028/1e5a8b07-946b-4de5-9971-b2872f1bf125)

### Figma design ⬇️  
![image](https://github.com/jovasal/jovasal.github.io/assets/117144028/f6a176d2-c6ef-476c-b85d-5329b79b7129)

frempongdev reviewed (approved):
# STATUS: APPROVED 🟢 ✔️  

Hi @jovasal , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media.giphy.com/media/ENagATV1Gr9eg/giphy.gif"/>

## Highlights
Professional Readme :heavy_check_mark: 
all application features work :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
